packages/utils/src/github/issuereporter.ts

```diff
@@ -24,8 +28,8 @@ export class IssueReporter {
 
     public constructor() {
         this.GITHUB_DATA = {
-            owner: this.GITHUB_OWNER,
-            repo: this.GITHUB_REPO
+            owner: this.GITHUB_OWNER!,
```

Comment:
What about defaulting to `''` in line 19-21 so we don't have to use `!` in this file?